### PR TITLE
Phase change patch 3: Changes to BrinkmanPenalization class and associated changes in the input files

### PR DESCRIPTION
**Pull request by amneetb**
Changes to BrinkmanPenalization class and associated changes in the input files. Very trivial change to a default value in Marangoni surface tension header file.

### IBAMR Pull Request Checklist
- [ ] Does the test suite pass?
- [ ] Was clang-format run?
- [ ] Were relevant issues cited? Please remember to add `Fixes #12345` to close
      the issue automatically if we are fixing the problem.
- [ ] Is this a change others will want to know about? If so, then has a
      changelog entry been added?
- [ ] If new data structures have been added to a class then have they been set
      up appropriately for restarts? If so, ensure that the restart version
      number is incremented.
- [ ] Does this change include a bug fix or new functionality? If so, a new test
      or tests should be added. New tests should run quickly (less than a minute
      in release mode). If possible, an older test should gain a new option so
      that we do not need to compile more test executables.
- [ ] Did you (if your account has permission to do so) set relevant labels on
      GitHub for the pull request?

### CODE REVIEW

File: examples/IBLevelSet/ex1/Makefile.am [hunk 1]
```diff
@@ -16,16 +16,13 @@ include $(top_srcdir)/config/Make-rules
 
 ## Dimension-dependent testers
 EXAMPLE_DRIVER = example.cpp SetFluidSolidDensity.cpp SetFluidSolidDensity.h SetFluidSolidViscosity.cpp SetFluidSolidViscosity.h TagLSRefinementCells.cpp TagLSRefinementCells.h
-EXTRA_DIST     = input2d
 
 EXAMPLES =
 EXTRA_PROGRAMS =
-if LIBMESH_ENABLED
```
Comment (drwells) [inline examples/IBLevelSet/ex1/Makefile.am:23]: Nice catch!

File: include/ibamr/MarangoniSurfaceTensionForceFunction.h [hunk 2]
```diff
@@ -191,7 +191,7 @@ class MarangoniSurfaceTensionForceFunction : public SurfaceTensionForceFunction
     /*!
      * Marangoni coefficient.
      */
-    double d_marangoni_coefficient = std::numeric_limits<double>::signaling_NaN();
+    double d_marangoni_coefficient = 0.0;
```
Comment (drwells) [inline include/ibamr/MarangoniSurfaceTensionForceFunction.h:194]: Is 0.0 a reasonable default value? If not we should keep this as the invalid value to be sure it is set later on.

Comment (Raamkrishnan) [reply include/ibamr/MarangoniSurfaceTensionForceFunction.h]: Thank you for pointing this out. Now  I initialize with std::numeric_limits<double>::signaling_NaN().

**Comment by Raamkrishnan** [comment]
> Nice work. I have one minor comment.
> 
> I'm ambivalent about changing the interface to `BrinkmanPenalization`: the new version seems a lot better but it isn't great that we break backwards compatibility. Could you add a changelog entry describing how users can convert existing input files to meet the new input file format?

I have added a changelog with the sample `BrinkmanPenalization` database used in the input file.

**Review by drwells — APPROVED** [review]

**Pull request merged**